Sources/Sentry/SentryThreadInspector.m

```diff
@@ -25,6 +25,11 @@ - (id)initWithStacktraceBuilder:(SentryStacktraceBuilder *)stacktraceBuilder
 }
 
 - (NSArray<SentryThread *> *)getCurrentThreads
+{
+    return [self getCurrentThreadsWithStackTrace:NO];
+}
+
+- (NSArray<SentryThread *> *)getCurrentThreadsWithStackTrace:(BOOL)forceStackTraces
```

Comment:
`l`: forceStackTraces is a bit confusing to me. Maybe we can rename it something like
```suggestion
- (NSArray<SentryThread *> *)getCurrentThreadsWithStackTrace:(BOOL)getAllStacktraces
```
or we could create two different methods. One with `getCurrentThreadWithStackTrace` and another one with `getAllThreadsWithStackTraces`.

Reply:
I will change the parameter and we still have a function without parameter that only get the stack trace for the current thread,